jenkins/ci.suse.de/cloud-crowbar9.yaml

```diff
@@ -41,6 +41,25 @@
     jobs:
         - '{crowbar_job}'
 
+- project:
+    name: cloud-crowbar9-job-upgrade-x86_64
+    crowbar_job: '{name}'
+    cloudsource: develcloud8
+    upgrade_cloudsource: develcloud9
+    cloud_env: cloud-crowbar-ci-slot
+    scenario_name: std-lmm
+    controllers: '1'
+    computes: '1'
+    ses_enabled: false
+    enabled_services: database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon
+    extra_params: |
+      ssl_enabled=false
+      upgrade_repos=http://download.suse.de/ibs/home:/jgrassler:/branches:/Devel:/Cloud:/9:/Staging/SLE_12_SP4/
```

Comment:
is this supposed to be there? upgrade to regular develcloud9staging should work..

Reply:
I removed `upgrade_repos`. The job upgrades from stagingcloud8 to stagingcloud8 now.